css/index.css

```diff
@@ -143,4 +210,141 @@ footer nav {
 footer nav a {
     color: white;
     text-decoration: none;
+}
+
+/*Header Nav*/
+
+li, a, button .logo {
+    font-size: 20px;
+    color: black;
+    text-decoration: none;
+    font-family: Georgia, 'Times New Roman', Times, serif;
+}
+
```

Comment:
Good job styling the header and spacing out the anchors with flexbox.  The image should go across the screen in like in the design file and after that the header/nav will be complete!